dracut/parse-kickstart

```diff
@@ -163,9 +163,7 @@ class DracutURL(Url, DracutArgsMixin):
 
 class DracutUpdates(Updates, DracutArgsMixin):
     def dracut_args(self, args, lineno, obj):
-        if self.url == "floppy":
-            return "live.updates=/dev/fd0"
-        elif self.url:
+        if self.url:
```

Comment:
What happens if "floppy" is there? Intentional error?

Reply:
There will be no officially supported way how to set `url` to `floppy`. Users were supposed to specify `updates` without any arguments to use the floppy disk and that won't be possible with the pykickstart update. It is possible to specify `updates floppy` in a kickstart file, but the documentation says that the argument should be a URL.

Reply:
OK, changing to ACK then.

I understand this as "a device can be there too but we are not telling users that". Which is fine.